module/documents/actor/actor.mjs

```diff
@@ -1507,16 +1519,24 @@ export default class Actor5e extends Actor {
    * @returns {Promise<void>}           A promise which resolves once initiative has been rolled for the Actor
    */
   async rollInitiativeDialog(rollOptions={}) {
+    let {advantageMode, fastForward, event} = rollOptions;
+    const ff = _determineAdvantageMode({fastForward, event});
+    if ( ff.isFF ) advantageMode ??= ff.advantageMode;
```

Comment:
I don't personally agree with fast-forwarding the initiative roll. Unlike other D20 rolls where Adv/Disadv are common, having those modifiers for initiative is very rare.  It's more likely to result in a mis-click that incorrectly applies adv/disadv to initiative than it is to save a user time. Furthermore, people aren't used to checking the chat log for initiative to ensure that their initiative was rolled properly, so an initiative roll that incorrectly has advantage or disadvantage is unlikely to be noticed.